packages/generator/__tests__/assets/templates/dynamicFontSizeVertical.json

```diff
@@ -58,7 +58,8 @@
         "fontName": "NotoSerifJP-Regular",
         "dynamicFontSize": {
           "min": 13,
-          "max": 36
+          "max": 36,
+          "fit": "vertical"
```

Comment:
I want to check that it works with the default value as "vertical" without specifying "fit".